doc/dev/mgmt_quickstart.md

```diff
@@ -0,0 +1,340 @@
+ResourceGroupCollection resourceGroups = subscription.GetResourceGroups();
+
+// With the collection, we can create a new resource group with an specific name
+string resourceGroupName = "myRgName";
+AzureLocation location = AzureLocation.WestUS2;
+ResourceGroupData resourceGroupData = new ResourceGroupData(location);
+ArmOperation<ResourceGroupResource> operation = await resourceGroups.CreateOrUpdateAsync(WaitUntil.Completed, resourceGroupName, resourceGroupData);
+ResourceGroupResource resourceGroup = operation.Value;
+```
+
+### List all resource groups
+```C# Snippet:Managing_Resource_Groups_ListAllResourceGroup
+// First, initialize the ArmClient and get the default subscription
+ArmClient client = new ArmClient(new DefaultAzureCredential());
+SubscriptionResource subscription = await client.GetDefaultSubscriptionAsync();
+// Now we get a ResourceGroupResource collection for that subscription
+ResourceGroupCollection resourceGroups = subscription.GetResourceGroups();
+// We can then iterate over this collection to get the resources in the collection
+await foreach (ResourceGroupResource resourceGroup in resourceGroups)
+{
+    Console.WriteLine(resourceGroup.Data.Name);
+}
+```
+
+### Update a resource group
+```C# Snippet:Managing_Resource_Groups_UpdateAResourceGroup
+// Note: Resource group named 'myRgName' should exist for this example to work.
+ArmClient client = new ArmClient(new DefaultAzureCredential());
+SubscriptionResource subscription = await client.GetDefaultSubscriptionAsync();
+ResourceGroupCollection resourceGroups = subscription.GetResourceGroups();
+string resourceGroupName = "myRgName";
+ResourceGroupResource resourceGroup = await resourceGroups.GetAsync(resourceGroupName);
+resourceGroup = await resourceGroup.AddTagAsync("key", "value");
+```
+
+### Delete a resource group
+```C# Snippet:Managing_Resource_Groups_DeleteResourceGroup
+ArmClient client = new ArmClient(new DefaultAzureCredential());
+SubscriptionResource subscription = await client.GetDefaultSubscriptionAsync();
+ResourceGroupCollection resourceGroups = subscription.GetResourceGroups();
+string resourceGroupName = "myRgName";
+ResourceGroupResource resourceGroup = await resourceGroups.GetAsync(resourceGroupName);
+await resourceGroup.DeleteAsync(WaitUntil.Completed);
+```
+
+## Code Samples
+
+More code samples for using the management library for .NET can be found in the following locations
+
+- [.NET Management Library Code Samples](https://docs.microsoft.com/samples/browse/?branch=master&languages=csharp&term=managing%20using%20Azure%20.NET%20SDK&terms=managing%20using%20Azure%20.NET%20SDK)
+
+## Need help?
+
+- File an issue via [Github
+  Issues](https://github.com/Azure/azure-sdk-for-net/issues) and
+  make sure you add the "Preview" label to the issue
+- Check [previous
+  questions](https://stackoverflow.com/questions/tagged/azure+.net)
+  or ask new ones on StackOverflow using azure and .NET tags.
+
+## Contributing
```

Comment:
Unfortunately we forgot to update the contributing section here in this [PR](https://github.com/Azure/azure-sdk-for-net/pull/27947), can we do it now?

Reply:
fixed